docs/src/main/asciidoc/keycloak-authorization-guide.adoc

```diff
@@ -153,12 +152,10 @@ public class AdminResource {
 Note that we did not define any annotation such as `@RoleAllowed` to explicitly enforce access to a resource.
 The extension will be responsible to map the URIs of the protected resources you have in Keycloak and evaluate the permissions accordingly, granting or denying access depending on the permissions that will be granted by Keycloak.
 
-== Configuring the application
+=== Configuring the application
```

Comment:
@pedroigor I've looked at the complete document, what might help a bit if, in the very top introduction, you mention somehow that this `quarkus-keycloak-authorization` extension complements `quarkus-oidc` as opposed to be possibly seen by some users as a completely independent extension. It is probably obvious enough from the configuration example, but IMHO it would be clearer if there was some clarification